Mathlib/CategoryTheory/DiscreteCategory.lean

```diff
@@ -183,9 +183,17 @@ theorem functor_obj {I : Type u₁} (F : I → C) (i : I) :
   rfl
 #align category_theory.discrete.functor_obj CategoryTheory.Discrete.functor_obj
 
+@[simp]
```

Comment:
This was simp 1100 in the other file - did you mean to change this?

Reply:
Yes, this change was intentional. I suspect the 1100 was only added to avoid the `simp can prove this` lint, but reordering this also solves it.